index.js

```diff
@@ -267,6 +267,40 @@ client.on("voiceStateUpdate", (oldState, newState) => {
             delete callRecords[userId];
         }
     }
+
+    // Check emojis for deleting todo:
+    const message = reaction.message.content;
+    const regex = /<@(\d+)>/g;
+
+    if (reaction.emoji.name === deleteTodoEmoji) {
+        if (message.match(regex)[0] === `<@${user.id}>`) {
+            deleteTodo[0] = true;
+        }
+        if (message.match(regex)[1] === `<@${user.id}>`) {
+            deleteTodo[1] = true;
+        }
+        if (deleteTodo[0] === true && deleteTodo[1] === true) {
+            reaction.message.delete();
+            deleteTodo[0] = false;
+            deleteTodo[1] = false;
```

Comment:
Note that this only works for 2 users at a time. You should create an object and then have the key be `messageId` and then the value be `[false, false]` so that this way it's per message ID. Otherwise, if two users are marking two different Todo's, then your `deleteTodo` will act like it's counting for both